explore/src/main/scala/explore/proposal/ProposalEditor.scala

```diff
@@ -151,13 +152,14 @@ object ProposalEditor:
               props.userIsReadonlyCoi
             ),
           (_, _) =>
-            Option
-              .unless[VdomNode](props.proposalOrUserIsReadonly):
-                <.div(
-                  ExploreStyles.AddProgramUserButton,
+            <.div(
+              ExploreStyles.AddProgramUserButton,
+              Option
+                .unless[VdomNode](props.proposalOrUserIsReadonly):
                   AddReadonlyCoiButton(props.programId, props.users)
-                )
-              .orEmpty
+                .orEmpty,
+              HelpIcon("proposal/main/investigators.md".refined)
```

Comment:
So, the idea is that even if the `proposalOrUserIsReadonly` you swapped this around to show the div anyway because you're putting in the help icon?

Reply:
Correct. I figure the help information might be useful, anyway, even if they can't add users.